Enhancement/pull first

ChristianKniep opened the pull request:
- changed the setup steps for the bash variables to make it more dense
- put $1/$2/$3 into more elaborate variables and used them throughout
- download `api garant jobrunner nginx notary-server notary-signer postgres registry rethink` dtr images before installing using `docker/dtr --install`
- switch the visualiser image from `manomarks/visualizer` to `dockersamples/visualizer`
- disabled healthcheck and named minio-service

ChristianKniep committed:
Minor changes to enhance robustness and use different viz-image

ChristianKniep committed:
cleaner default value setup

ChristianKniep commented:
Oh wait, did I honor your UCP/DTR version choice?

ChristianKniep committed:
honor DTRVERSION choice

foodebeer reviewed (commented):
Hey @qnib, I see 2 different environment vars: DTR_TAG and DTRVERSION. Is that intentional?

ChristianKniep commented:
I introduced in my first iteration `DTR_VER`, but that should be gone in the final, rebased version.
```
$ grep DTR_ *
setupddc.sh:DTR_CNT=${2:-0}
setupddc.sh:if [[ ${DTR_CNT} == 0 ]]; then
setupddc.sh:if [ ${DTR_CNT} == 0 ]; then
setupddc.sh:    for (( COUNT=1; COUNT \< ${DTR_CNT}; COUNT++))
```
